helpers/helpers_test.go

```diff
@@ -0,0 +1,107 @@
+package helpers
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type TestSuite struct {
+	suite.Suite
+}
+
+func TestConfigTestSuite(t *testing.T) {
+	suite.Run(t, new(TestSuite))
+}
+
+func (suite *TestSuite) SetupTest() {
+
+}
+
+func (suite *TestSuite) TestCreateErrorResponse() {
+
+	errorMessage := "some error"
+	errorBytes := CreateErrorResponse(errorMessage)
+
+	assert.Equal(suite.T(), errorBytes, []byte("{\"error\":{\"message\":\"some error\"}}"))
+}
+
+func (suite *TestSuite) TestNewConf() {
+	confData := `global:
+  iss: "https://some.url"
+  pathToKey: "../dev_utils/dummy.ec.nbis.se"
```

Comment:
better to have an init function that generates the key in a temp folder that will be used by all test functions that requires the key
```suggestion
  pathToKey: ""
```